addons/common/functions/fnc_hasHatch.sqf

```diff
@@ -0,0 +1,35 @@
+/*
+ * Author: commy2
+ * Check if unit has a hatch and can turn out.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Unit has a hatch? <BOOL>
+ *
+ * Public: Yes
+ */
+#include "script_component.hpp"
+
+params ["_unit"];
+
+local _vehicle = vehicle _unit;
```

Comment:
There is regex ready to change it, it's still `local` until 1.54 gets rolled and that won't happen for at least 2 more weeks.